modules/monitoring-modifying-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-core-platform-alerting-rules_{context}"]
+= Modifying core platform alerting rules
+
+As a cluster administrator, you can modify core platform alerts before Alertmanager routes them to a receiver. 
+For example, you can change the severity label of an alert, add a custom label, or exclude an alert from being sent to Alertmanager.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have enabled Technology Preview features, and all nodes in the cluster are ready.
+
+
+.Procedure
+
+. Create a new YAML configuration file named `example-modified-alerting-rule.yaml` in the `openshift-monitoring` namespace.
+
+. Add an `AlertRelabelConfig` resource to the YAML file. 
+The following example modifies the `severity` setting to `critical` for the default platform `watchdog` alerting rule: 
++
+[source,yaml]
+----
+apiVersion: monitoring.openshift.io/v1alpha1
+kind: AlertRelabelConfig
+metadata:
+  name: watchdog
+  namespace: openshift-monitoring
+spec:
+  configs:
+  - sourceLabels: [alertname,severity] <1>
+    regex: "Watchdog;none" <2>
+    targetLabel: severity <3>
+    replacement: critical <4>
+    action: Replace <5>
+----
+<1> The source labels for the values to be modified.
+<2> The name of the alerting rule you want to modify.
```

Comment:
according to https://github.com/openshift/api/blob/master/monitoring/v1alpha1/0000_50_monitoring_02_alertrelabelconfigs.crd.yaml#L64-L66
we'd better change to:
regex against which the value of source labels is matched.

Reply:
yeah good catch!

Reply:
Changed to:

`The relabel action that replaces the old value with the new value. The new value is based on the regex against which the value of the source labels is matched.`